superdesk/users/users.py

```diff
@@ -45,22 +44,20 @@ def __init__(self, endpoint_name, app, service, endpoint_schema=None):
                 'readonly': self.readonly
             },
             'display_name': {
-                'type': 'string',
-                'readonly': self.readonly
+                'type': 'string'
             },
             'email': {
                 'unique': True,
                 'type': 'email',
-                'required': True
+                'required': True,
+                'readonly': self.readonly
```

Comment:
hmm yeah this might be a problem, it was ok for me but I should test bringing in a new user via LDAP